src/data/nat/digits.lean

```diff
@@ -317,6 +318,23 @@ function.left_inverse.injective (of_digits_digits b)
   b.digits n = b.digits m ↔ n = m :=
 (digits.injective b).eq_iff
 
+lemma length_digits (b n : ℕ) (hb : 2 ≤ b) (hn : 0 < n) :
```

Comment:
seems elsewhere in the file they go for `digits_len`; I like this name more personally but it's probably worth having a standard one in the file